Resources/Prototypes/_Mono/Entities/Clothing/Back/backpacks.yml

```
@@ -0,0 +1,28 @@
+  parent: ClothingBackpack
+  id: ClothingBackpackPerogee
+  name: perogee-pattern backpack
+  description: Despite the name and looks, its just a normal backpack. Dissapointing.
+  components:
+  - type: Sprite
+    sprite: _Mono/Clothing/Back/Backpacks/advanced.rsi
+  - type: ExplosionResistance
+    damageCoefficient: 0.1
+  - type: Storage
+    grid:
+    - 0,0,8,4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 5

(indentation)